### ISSUE TITLE
Screencast url broken

### ISSUE THREAD

**Issue by wdebusschere** [DeuxHuitHuit/entry_relationship_field#78]
https://www.screenr.com/pDDN

**Comment by nitriques** [comment]
Yeah I know, screenr is  now dead and I can not find the video elsewhere. We need to redo it.

cc @alexnantel88

**Comment by swiegmann** [comment]
@nitriques,
Here you go ;)

[Screenr - entry relation ship Symphony CMS.zip](https://github.com/DeuxHuitHuit/entry_relationship_field/files/2517328/Screenr.-.entry.relation.ship.Symphony.CMS.zip)
